server/service/src/sync/program_schemas/hiv_testing_encounter.json

```diff
@@ -160,7 +172,7 @@
           },
           "type": "object"
         },
-        "notes": {
+        "note": {
```

Comment:
This should really be `notes` since it is an array could you please update the base encounter type in the schema repo to fix this? @CarlosNZ this might conflict with your work?

Reply:
[e8a929e](https://github.com/openmsupply/open-msupply/pull/1221/commits/e8a929e50472b588f730e3a35e88d6715bed9043)